src/SIL.XForge.Scripture/ClientApp/e2e/src/login.po.ts

```diff
@@ -29,8 +29,14 @@ export class LoginPage {
     await this.loginButton.click();
 
     // wait for redirect before enabling angular wait
-    await browser.sleep(300);
-    browser.waitForAngularEnabled(true);
+    //await browser.sleep(300);
+    // browser.waitForAngularEnabled(true);
+    browser.wait(() => {
+      browser.waitForAngularEnabled(false);
+      return browser.getCurrentUrl().then(() => {
+          browser.waitForAngularEnabled(true);
+      });
+    }, 5000);
```

Comment:
Use a test constant, e.g. `this.constants.conditionTimeout`.

Reply:
Okay, we have changed into test constant as per "this.constants.conditionTimeout". Thank you.